src/Home/index.tsx

```diff
@@ -82,198 +86,263 @@ const Home: FunctionComponent = () => {
-            {displayCovidData && <COVIDDataDashboard />}
+          <View style={style.statusBottomContainer}>
+            <Text style={style.statusActionText}>{actionText}</Text>
+            <SvgXml
+              xml={Icons.ChevronRight}
+              fill={Colors.black}
+              width={Iconography.tiny}
+              height={Iconography.tiny}
+            />
           </View>
-          <View style={style.ctaContainer}>
+        </TouchableOpacity>
+        <ShareLink />
+        {displayCovidData && <COVIDDataDashboard />}
+        {displayCallbackForm && (
+          <View style={style.floatingContainer}>
+            <Text style={style.sectionHeaderText}>
+              {t("home.did_you_test_positive")}
+            </Text>
+            <Text style={style.sectionBodyText}>
+              {t("home.to_submit_your_test")}
+            </Text>
             <Button
-              onPress={handleOnPressReportTestResult}
-              label={t("home.bluetooth.report_positive_result")}
+              onPress={handleOnPressTalkToContactTracer}
+              label={t("home.talk_to_a_contact")}
               customButtonStyle={style.button}
               customButtonInnerStyle={style.buttonInner}
               hasRightArrow
             />
-            {displaySelfAssessment && (
-              <Button
-                onPress={handleOnPressTakeSelfAssessment}
-                label={t("home.bluetooth.take_self_assessment")}
-                customButtonStyle={{
-                  ...style.button,
-                  ...style.selfAssessmentButton,
-                }}
-                customButtonInnerStyle={style.buttonInner}
-                outlined
-              />
-            )}
           </View>
+        )}
+        <View style={style.floatingContainer}>
+          <Text style={style.sectionHeaderText}>
+            {t("home.have_a_positive_test")}
+          </Text>
+          <Text style={style.sectionBodyText}>
+            {t("home.if_you_have_a_code")}
+          </Text>
+          <Button
+            onPress={handleOnPressReportTestResult}
+            label={t("home.submit_test_result_code")}
+            customButtonStyle={style.button}
+            customButtonInnerStyle={style.buttonInner}
+            hasRightArrow
+          />
         </View>
+        {displaySelfAssessment && (
+          <View style={style.floatingContainer}>
+            <Text style={style.sectionHeaderText}>
+              {t("home.feeling_sick")}
+            </Text>
+            <Text style={style.sectionBodyText}>
+              {t("home.check_if_your_symptoms")}
+            </Text>
+            <Button
+              onPress={handleOnPressTakeSelfAssessment}
+              label={t("home.bluetooth.take_self_assessment")}
+              customButtonStyle={style.button}
+              customButtonInnerStyle={style.buttonInner}
+            />
+          </View>
+        )}
+        <TouchableOpacity
```

Comment:
What do you think about `<CallEmergencyNumber />`

Reply:
👍